google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/QueryParameterValue.java

```diff
@@ -60,6 +62,21 @@
 @AutoValue
 public abstract class QueryParameterValue implements Serializable {
 
+  private static final DateTimeFormatter timestampValidator =
```

Comment:
Can we replace `timestampFormatter` with your configuration for `timestampValidator`?

Reply:
It can't be done with single formatter.